update-config-files.php

```diff
@@ -0,0 +1,369 @@
+   */
+    function writePortalConfig()
+    {
+        $portalConfig = array(
+          'dbHost' => array('varName' => 'dbHost', 'varValue' => '', 'varComment' => '', 'varType' => ''),
+          'dbName' => array('varName' => 'dbName', 'varValue' => '', 'varComment' => '', 'varType' => ''),
+          'dbUser' => array('varName' => 'dbUser', 'varValue' => '', 'varComment' => '', 'varType' => ''),
+          'dbPass' => array('varName' => 'dbPass', 'varValue' => '', 'varComment' => '', 'varType' => ''),
+
+          'title' => array('varName' => 'title', 'varValue' => '', 'varComment' => '', 'varType' => ''),
+          'city' => array('varName' => 'city', 'varValue' => '', 'varComment' => ''),
+          'adminLogonName' => array('varName' => 'adminLogonName', 'varValue' => '', 'varComment' => 'Administrator\'s logon name', 'varType' => ''),
+          'adPath' => array('varName' => 'adPath', 'varValue' => '', 'varComment' => 'Active directory path', 'varType' => ''),
+          'uploadDir' => array('varName' => 'uploadDir', 'varValue' => '', 'varComment' => 'Directory for user uploads. using backslashes (/), with trailing slash', 'varType' => ''),
+          'leafSecure' => array('varName' => 'leafSecure', 'varValue' => '', 'varComment' => '', 'varType' => ''),
+          'orgchartPath' => array('varName' => 'orgchartPath', 'varValue' => '', 'varComment' => 'HTTP Path to orgchart with no trailing slash', 'varType' => ''),
+          'orgchartImportTags' => array('varName' => 'orgchartImportTags', 'varValue' => "array('')", 'varComment' => 'Import org chart groups if they match these tags', 'varType' => ''),
+          'descriptionID' => array('varName' => 'descriptionID', 'varValue' => '', 'varComment' => 'indicator ID for description field', 'varType' => ''),
+          'emailPrefix' => array('varName' => 'emailPrefix', 'varValue' => '', 'varComment' => 'Email prefix', 'varType' => ''),
+          'emailCC' => array('varName' => 'emailCC', 'varValue' => '', 'varComment' => 'CCed for every email', 'varType' => ''),
+          'emailBCC' => array('varName' => 'emailBCC', 'varValue' => '', 'varComment' => 'BCCed for every email', 'varType' => ''),
+          'phonedbHost' => array('varName' => 'phonedbHost', 'varValue' => '', 'varComment' => '', 'varType' => ''),
+          'phonedbName' => array('varName' => 'phonedbName', 'varValue' => '', 'varComment' => '', 'varType' => ''),
+          'phonedbUser' => array('varName' => 'phonedbUser', 'varValue' => '', 'varComment' => '', 'varType' => ''),
+          'phonedbPass' => array('varName' => 'phonedbPass', 'varValue' => '', 'varComment' => '', 'varType' => ''),
+          'myNewVar' => array('varName' => 'myNewVar', 'varValue' => "'myNewValue'", 'varComment' => 'my new comment', 'varType' => 'static'),
+        );
+        $portalConfigs = rGlob('./*config.php');
+        foreach ($portalConfigs as $config)
+        {
+            if (preg_match('/\\/LEAF_Request_Portal\\/\\w.*/', $config))
```

Comment:
The folder won't necessarily be named LEAF_Request_Portal. Should be a way to find the folder name based off the location of db_config.php

Reply:
done!